### ISSUE TITLE
"Abnormal CSF amyloid level"

### ISSUE THREAD

**Issue by slaulederkind** [obophenotype/human-phenotype-ontology#901]
I would like to see "Abnormal CSF amyloid level" added to the HP ontology under the parent term "Abnormality of the cerebrospinal fluid" (HP:0002921).  The definition could be "Abnormal concentration of amyloid in the cerebrospinal fluid (CSF).".  An example can be found in PMID:12824768. A synonym would be "Abnormal CSF A[beta]42 level".

**Comment by cmungall** [comment]
Can you suggest a PRO ID for 'amyloid'?

**Comment by slaulederkind** [comment]
Here are appropriate PRO IDs by species: PR:P08592 (rat), PR:P12023 (mouse), PR:000004168 (human)

**Comment by slaulederkind** [comment]
See comment in GitHub:

PR:P08592 (rat), PR:P12023 (mouse), PR:000004168 (human)

Stanley J. F. Laulederkind, Ph.D.
Rat Genome Database
Human Molecular Genetics Center
Medical College of Wisconsin
8701 Watertown Plank Rd
Milwaukee WI 53226
(Phone) 414-955-7513
(Fax) 414-456-6516

**Comment by pnrobinson** [comment]
Am adding this term. Added also the child terms (elevated, decreased). I used following comment

In Alzheimer disease, histopathology typically reveals extracellular accumulations of amyloid beta (Abeta) in senile plaques and intracellular neurofibrillary tangles of hyperphosphorylated tau (P-tau). A[beta]42 is the 42 amino acid isoform of amyloid beta. Using positron emission tomography several studies found an inverse correlation between brain amyloid load and CSF A[beta]42.
